src/core/server/server.ts

```diff
@@ -291,7 +294,7 @@ export class Server {
 
   public async start() {
     this.log.debug('starting server');
-    const startTransaction = apm.startTransaction('server_start', 'kibana_platform');
+    const startTransaction = apm.startTransaction('server-start', kibana-platform);
```

Comment:
Is it worth adding labels to these transactions? An example would be to add the list of plugins that are enabled and we can measure the duration of the start based on how many plugins are active. Just a thought and not blocking for this PR.

Reply:
It will be a rare case to run Kibana with disabled plugins - starting from `v8.0` only a handful of plugins can be disabled. I'm worried it adds a lot of noise to the APM data since Kibana has > 120 plugins.

Reply:
I agree with @mshustov 
If we decided we want additional labels later, we can add them